docs/faq/groups.md

```diff
@@ -10,51 +10,58 @@ description: Player Grouping Functionality in Music Assistant
     
 ## Manual Sync
 
-These are temporary group players that are configured via the [Player List](ui.md#player-list).
+These are temporary group players that are configured via the [Player List](ui.md#player-list) or HA action. They provide a very flexible way to create and destroy groups as the players can be added or removed at any time. 
```

Comment:
Do we want to call this group players ?
Because strictly taken that is not the case. Its a single player which has some other players synced to the same content. So maybe a "temporary sync group" is the better wording ?

Reply:
Have a look at the latets changes to see if that is what you meant. Temporary Sync Groups make sense as they are very close to the same as a Sync Group with dynamic members option. I will change the UI page once we agree on the term